app/screens/Account/surveyChoose.js

```diff
@@ -1,69 +1,59 @@
 import React, { Component } from "react";
-import {
-  StyleSheet,
-  View,
-  Button,
-  TouchableOpacity,
-  Modal,
-  TouchableHighlight
-} from "react-native";
+import { View, TouchableOpacity, Modal } from "react-native";
 import { NavigationActions } from "react-navigation";
 import Icon from "react-native-vector-icons/MaterialIcons";
+import { connect } from "react-redux";
 
-import { Text, Wrapper } from "../../components";
+import {
+  clearRemoteSurveys,
+  fetchRemoteSurvey,
+  listRemoteSurveys
+} from "../../actions";
+import { StatusBar, Text } from "../../components";
+import RemoteSurveyList from "./RemoteSurveyList";
+import { selectRemoteSurveys } from "../../selectors";
 import { baseStyles } from "../../styles";
 
 class SurveyModal extends Component {
-  state = {
-    modalVisible: true
-  };
+  onBackPress = () => this.props.navigation.dispatch(NavigationActions.back());
 
-  setModalVisible(visible) {
```

Comment:
I wasn't sure how to deal with this. Since it was handled by react-navigation, it didn't seem like it was actually behaving as a modal. That explains why transparency didn't seem to be working as expected.

I can address this post-merge and turn it into a proper modal rather than a full-blown screen.